src/libs/ajax/leonardo/models/app-models.ts

```diff
@@ -25,6 +25,34 @@ export type AppStatus =
   | 'STOPPED'
   | 'STARTING';
 
+export type DisplayAppStatus =
+  | 'Running'
+  | 'Deleted'
+  | 'Deleting'
+  | 'Creating'
+  | 'Resuming'
+  | 'Status_unspecified'
+  | 'Error'
+  | 'Pausing'
+  | 'Paused';
+
+export interface AppStatusObject {
+  status: AppStatus;
+  displayStatus: DisplayAppStatus;
```

Comment:
AppStatusDisplay?
and then change displayStatus -> statusDisplay.
I think that's consistent elsewhere?

Reply:
it's not but I like your way better, the way you explain it